models/intermediate/int_qualtrics__survey.sql

```diff
@@ -62,7 +62,7 @@ survey_join as (
     left join user as creator
         on survey.creator_user_id = creator.user_id
         and survey.source_relation = creator.source_relation
-    join agg_versions 
+    join agg_versions
```

Comment:
Same comment about explicitly defining the type of join.

Reply:
updated to left